Gruntfile.js

```diff
@@ -65,8 +64,14 @@ module.exports = function(grunt) {
           'modules/reliability/js/columnFormatter.js': 'modules/reliability/jsx/columnFormatter.js'
         }
       }
+    },
+
+    run: {
+      webpack: {
+        cmd: 'webpack'
```

Comment:
@jacobpenny I was using local grunt, but running your command results in the same error.